platform/viewer/public/theme/theme.json5

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
is this intentionally empty? can we have an actual hp here so that the demo can showcase the `confi-point` stuff?